token.in

```diff
@@ -0,0 +1,30 @@
++ 
+- 
+* 
+/ 
+:== 
+< 
+<:= 
+:= 
+>:=
+
+{     
+}  
+: 
+; 
+space
+
+let
+const
+char
+do 
+else 
+if 
+int 
+readFromKb 
+writeToScr 
+readFromFile 
+writeToFile 
```

Comment:
maybe some keywords could be less verbose, such as "read" instead of readFromKb or "write" instead of "writeToScr". For files, maybe "fread" and "fwrite"